component/src/main/java/com/synopsys/integration/alert/component/users/web/role/RoleActions.java

```diff
@@ -113,21 +112,21 @@ public RoleActions(UserManagementDescriptorKey userManagementDescriptorKey, Role
     }
 
     @Override
-    protected ActionResponse<MultiRolePermissionModel> readAllWithoutChecks() {
-        List<RolePermissionModel> roles = roleAccessor.getRoles().stream()
-                                              .map(this::convertUserRoleModel)
-                                              .collect(Collectors.toList());
-        return new ActionResponse<>(HttpStatus.OK, new MultiRolePermissionModel(roles));
+    protected List<UserRoleModel> getDatabaseModels() {
+        return new ArrayList<>(roleAccessor.getRoles());
     }
 
     @Override
-    protected ActionResponse<RolePermissionModel> readWithoutChecks(Long id) {
-        Optional<RolePermissionModel> role = findExisting(id);
-        if (role.isPresent()) {
-            return new ActionResponse<>(HttpStatus.OK, role.get());
-        }
-        //This is covered by the findExistingCheck in AbstractResourceActions. TODO for 6.4.0
-        return new ActionResponse<>(HttpStatus.NOT_FOUND, String.format("Role with id:%d not found.", id));
+    protected MultiRolePermissionModel createMultiResponseModel(final List<RolePermissionModel> roles) {
+        return new MultiRolePermissionModel(roles);
+    }
+
+    @Override
+    protected RolePermissionModel convertDatabaseModel(UserRoleModel userRoleModel) {
```

Comment:
I think it would be more clear if the method name included the result. e.g. `convertDatabaseModelToRestModel` or something along those lines.

Reply:
Righto.